spec/jobs/close_petitions_job_spec.rb

```diff
@@ -28,7 +28,9 @@
       let(:now) { "2015-12-29T07:00:00Z".in_time_zone }
 
       it "does change the petition debate state" do
-        expect{
+        petition.update(closed_at: now)
```

Comment:
I'm thinking we should remove this job and spec to make the logic clearer like the refer or reject job